sdk/typescript/src/types/transactions.ts

```diff
@@ -48,7 +48,9 @@ export type SuiTransactionData = {
 
 // TODO: support u64
 export type EpochId = number;
```

Comment:
need additional changes in this file.

need to update `TransactionKindName` and `SuiTransactionKind` to include Pay.

something like:

```
export type SuiPay = {
  recipients: SuiAddress[];
  inputCoins: SuiObjectRef[];
  amounts: number[];
};

export type TransactionKindName =
  | 'TransferObject'
  | 'Publish'
  | 'Call'
  | 'TransferSui'
  | 'ChangeEpoch'
  | 'Pay';

export type SuiTransactionKind =
  | { TransferObject: TransferObject }
  | { Publish: SuiMovePackage }
  | { Call: MoveCall }
  | { TransferSui: SuiTransferSui }
  | { ChangeEpoch: SuiChangeEpoch }
  | { Pay: SuiPay };
```